requirements_dev.txt

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
We split requirements.txt up into prod and dev requirements. People installing client-python just to use it, only need the prod requirements (requirements.txt). Developers like us need the dev ones.